docs/design/contextual-tabs.md

```diff
@@ -358,7 +358,59 @@ function myContextChanges() {
 }
 ```
 
-## Error handling
+## Localizing the JSON blob
+
+The JSON blob that is passed to `requestCreateControls` is not localized the same way that the manifest markup for custom core tabs is localized as described at [Control localization from the manifest](../develop/localization.md#control-localization-from-the-manifest). Instead, the localization must occur at runtime using distinct JSON blobs for each locale. We suggest that you use a `switch` statement that tests the [Office.context.displayLanguage](/javascript/api/office/office.context#displayLanguage) property. The following is an example:
```

Comment:
The first sentence is a little confusing to me, but I think it could be resolved with additional punctuation. One idea: 

```suggestion
The JSON blob that is passed to `requestCreateControls` is not localized the same way that the manifest markup for custom core tabs is localized (which is described at [Control localization from the manifest](../develop/localization.md#control-localization-from-the-manifest)). Instead, the localization must occur at runtime using distinct JSON blobs for each locale. We suggest that you use a `switch` statement that tests the [Office.context.displayLanguage](/javascript/api/office/office.context#displayLanguage) property. The following is an example:
```